### PR TITLE
My app which uses Fresco for image loading.

### PR DESCRIPTION
**Pull request by mik-dass**
My app has been using Fresco library for image loading since day 1 of launch. Gr8 library ....:)

### CODE REVIEW

**Comment by facebook-github-bot-8** [comment]
Thank you for your pull request and welcome to our community. We require contributors to sign our Contributor License Agreement, and we don't seem to have you on file. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla - and if you have received this in error or have any questions, please drop us a line at cla@fb.com. Thanks!

**Comment by facebook-github-bot-9** [comment]
Thank you for signing our Contributor License Agreement. We can now accept your code for this (and any) Facebook open source project. Thanks!

File: _data/powered_by.yml [hunk 1]
```diff
@@ -68,3 +68,5 @@
     url: https://play.google.com/store/apps/details?id=com.sgiggle.production
   - name: CondomCraze
     url: https://play.google.com/store/apps/details?id=com.condomcraze.android
+  - name: memeham
+   url:  https://play.google.com/store/apps/details?id=com.memeham.beyourself.memeham
```
Comment (oprisnik) [inline _data/powered_by.yml]: please fix the indentation

**Comment by oprisnik** [comment]
See #1419

**Pull request closed without merging**